reactor-netty-core/build.gradle

```diff
@@ -106,8 +106,6 @@ dependencies {
 	compileOnly "io.micrometer:micrometer-core:$micrometerVersion"
 	compileOnly "io.micrometer:micrometer-tracing:$micrometerTracingVersion"
```

Comment:
Actually `micrometer-tracing` has a compile dependency on context-propagation

Reply:
Isn't it `implementation` and not `api`?

Reply:
It's `implementation`